### DISCUSSION TITLE
Vercel Links not working outside my country.

### DISCUSSION THREAD

**Discussion by kamiomar** [vercel/community#4964] [Help] [answered]
### Summary

Hi,

i have created my account from Pakistan and  deploy my code. 

https://halalla-staging.vercel.app/
https://admin-halalla.vercel.app/

these both link not working outside Pakistan. i have tried it from UAE and USA. 

### Steps to Reproduce

https://halalla-staging.vercel.app/
https://admin-halalla.vercel.app/

the above mention link not working outside Pakistan.

**Comment by swarnava** [comment, marked as answer]
Hi, 

I just tried to access it in India and using VPN from different country, they seems opening fine. What is the error you are receiving?

**Reply by kamiomar** [reply]
![screencapture-halalla-staging-vercel-app-2023-11-25-18_08_27](https://github.com/vercel/community/assets/522984/fd3ebe83-53c0-4569-a693-54cb99e602c3)

**Reply by kamiomar** [reply]
i have used the usa vpn.
